cpp/src/slice2js/Gen.cpp

```diff
@@ -2762,7 +2762,8 @@ Slice::Gen::TypeScriptVisitor::visitEnum(const EnumPtr& p)
     _out << nl << " * Returns the enumerator for the given value.";
     _out << nl << " *";
     _out << nl << " * @param value The enumerator value.";
-    _out << nl << " * @returns The enumerator for the given value.";
+    _out << nl << " * @returns The enumerator for the given value or undefined if the value does not correspond to any";
+    _out << nl << " * enumerator.";
     _out << nl << " */";
     _out << nl << "static valueOf(value: number): " << name << ";";
```

Comment:
The return type of this signature should probably be updated now right?
```suggestion
    _out << nl << "static valueOf(value: number): " << name << " | undefined;";
```